src/main.cpp

```diff
@@ -1,13 +1,7 @@
 #include "stdafx.h"
-#define WITH_GUI 1
```

Comment:
i would like to keep that line for those who want GUI.

Reply:
Why? This variable can be set from the compiler.

target_compile_definitions(polyvector_thing PRIVATE WITH_GUI=1)

As it already works. By default with GUI